// @sap/ux-specification/dist/src/specification/common/types.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var FlexibleColumnLayoutType;
(function (FlexibleColumnLayoutType) {
    FlexibleColumnLayoutType["OneColumn"] = "OneColumn";
    FlexibleColumnLayoutType["TwoColumnsBeginExpanded"] = "TwoColumnsBeginExpanded";
    FlexibleColumnLayoutType["TwoColumnsMidExpanded"] = "TwoColumnsMidExpanded";
    FlexibleColumnLayoutType["MidColumnFullScreen"] = "MidColumnFullScreen";
    FlexibleColumnLayoutType["ThreeColumnsMidExpanded"] = "ThreeColumnsMidExpanded";
    FlexibleColumnLayoutType["ThreeColumnsEndExpanded"] = "ThreeColumnsEndExpanded";
    FlexibleColumnLayoutType["ThreeColumnsMidExpandedEndHidden"] = "ThreeColumnsMidExpandedEndHidden";
    FlexibleColumnLayoutType["ThreeColumnsBeginExpandedEndHidden"] = "ThreeColumnsBeginExpandedEndHidden";
    FlexibleColumnLayoutType["EndColumnFullScreen"] = "EndColumnFullScreen";
})(FlexibleColumnLayoutType = exports.FlexibleColumnLayoutType || (exports.FlexibleColumnLayoutType = {}));
var FlexChangeLayer;
(function (FlexChangeLayer) {
    FlexChangeLayer["Vendor"] = "VENDOR";
    FlexChangeLayer["Customer"] = "CUSTOMER_BASE";
})(FlexChangeLayer = exports.FlexChangeLayer || (exports.FlexChangeLayer = {}));
var ExportType;
(function (ExportType) {
    ExportType["GW"] = "GW";
    ExportType["UI5Client"] = "UI5Client";
})(ExportType = exports.ExportType || (exports.ExportType = {}));
